packages/mock/tests/mocking-compatibility.spec.ts

```diff
@@ -80,19 +87,6 @@ describe('Mock retro-compatibility', () => {
     }
   `;
 
-  const resolveFunctions = {
```

Comment:
This also seems to be pre-v8: https://www.graphql-tools.com/docs/mocking/#breaking-change-preserved-resolvers-source-argument-and-abstract-types-mocking